Cargo.toml

```diff
@@ -33,6 +33,7 @@ chariott-common = { workspace = true }
 prost = { workspace = true }
 prost-types = { workspace = true }
 tokio = { workspace = true, features = ["rt-multi-thread"] }
+tokio-util = { workspace = true }
 tonic = { workspace = true }
 tonic-reflection = "0.5"
 tracing = { workspace = true }
```

Comment:
Can we make sure to [exclude the new E2E test from the test harness](https://github.com/eclipse/chariott/tree/main/tests#adding-new-tests)? Otherwise it's run when we only want to run unit tests.

Reply:
Done with e483bdbba2abaa572a657a16222d974d48118742.